plugins/modules/content_upload.py

```diff
@@ -166,10 +190,15 @@ def main():
                 size = os.stat(module.foreman_params['src']).st_size
                 content_upload_payload = {'size': size}
                 content_upload_payload.update(repository_scope)
+
                 content_upload = module.resource_action('content_uploads', 'create', content_upload_payload)
                 content_upload_scope = {'id': content_upload['upload_id']}
                 content_upload_scope.update(repository_scope)
 
+                if module.foreman_params['repository']['content_type'] == 'ostree':
+                    ostree_parameters = {'ostree_repository_name': ostree_repository_name, 'content_type': 'ostree_ref'}
+                    content_upload_scope.update(ostree_parameters)
```

Comment:
@jlsherrill do I read the Katello API correctly, we need to pass `ostree_repository_name` to the `import_uploads` call, but not the individual `content_uploads.update` calls? Aka: this hunk here is unnecessary? Test upload seems to pass fine without on a 4.3 box, but I wanted to be super sure.

Reply:
correct, the content_uploads api doesn't even know about this parameter